### PR TITLE
[Excel] (quick start) Add for content add-in

### PR DESCRIPTION
**Pull request by ElizabethSamuel-MSFT**

### CODE REVIEW

**Review by lindalu-MSFT — APPROVED** [review]
Looks good, Elizabeth

File: docs/quickstarts/excel-quickstart-content.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+### Create the add-in project
+
+1. In Visual Studio, choose **Create a new project**.
+
+1. Using the search box, enter **add-in**. Choose **Excel Web Add-in**, then select **Next**.
+
+1. Name your project **ExcelWebAddIn1** and select **Create**.
+
+1. In the **Create Office Add-in** dialog window, choose **Insert content into Excel spreadsheets** add-in type, then choose **Next**.
```
Comment (alison-mk) [inline docs/quickstarts/excel-quickstart-content.md:27]: This is minor, but I think including the article here will help with localization 
```suggestion
1. In the **Create Office Add-in** dialog window, choose the **Insert content into Excel spreadsheets** add-in type, then choose **Next**.
```

File: docs/quickstarts/excel-quickstart-content.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+1. Choose **Basic Add-in** or **Document Visualization Add-in** add-in template, and then choose **Finish** to create the project.
```
Comment (alison-mk) [inline docs/quickstarts/excel-quickstart-content.md:29]: Same reason as preceding comment
```suggestion
1. Choose the **Basic Add-in** or **Document Visualization Add-in** add-in template, and then choose **Finish** to create the project.
```

File: docs/quickstarts/excel-quickstart-content.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+
+1. In **Solution Explorer**, go to the **ExcelWebAddIn1** add-in project and open the **ExcelWebAddIn1Manifest** directory. This directory contains your manifest file, **ExcelWebAddIn1.xml**. The XML manifest file defines the add-in's settings and capabilities. See the preceding section [Explore the Visual Studio solution](#explore-the-visual-studio-solution) for more information about the two projects created by your Visual Studio solution.
+
+1. The `ProviderName` element has a placeholder value. Replace it with your name.
+
+1. The `DefaultValue` attribute of the `DisplayName` element has a placeholder. Replace it with **My Office Add-in**.
+
+1. The `DefaultValue` attribute of the `Description` element has a placeholder. Replace it with **A content add-in for Excel.**.
+
+1. Save the file.
```
Comment (alison-mk) [inline docs/quickstarts/excel-quickstart-content.md:47]: Should we include a note that explains what the code sample shows? 
```suggestion
1. Save the file. The updated lines should look like the following code sample.
```

**Review by alison-mk — APPROVED** [review]
Approved with a couple suggestions

**Review by lindalu-MSFT — COMMENTED** [review]
Looks good to me!

**Pull request merged**